assets/scripts/characters/mario/Mario.ts

```diff
@@ -0,0 +1,378 @@
+// Learn TypeScript:
+//  - https://docs.cocos.com/creator/manual/en/scripting/typescript.html
+// Learn Attribute:
+//  - https://docs.cocos.com/creator/manual/en/scripting/reference/attributes.html
+// Learn life-cycle callbacks:
+//  - https://docs.cocos.com/creator/manual/en/scripting/life-cycle-callbacks.html
+
+import Character from '../Character';
+import DIRECTION, { DIRECTIONS } from '../DIRECTION';
+import IDyingCharacter from '../IDyingCharacter';
+import MARIO_STATES from './MARIO_STATES';
+import MOVEMENT_TYPE from './MOVEMENT_TYPE';
+
+const { ccclass } = cc._decorator;
+
+@ccclass
+export default class Mario extends Character implements IDyingCharacter {
+  private isAlive = true;
+
+  public jumpForce = 9000;
+
+  public moveForce = 600;
+
+  public maxSpeed = 200;
+
+  public walkingMaxSpeed = 200;
```

Comment:
Why not property?

Reply:
I don't want this to be a property. Mario won't change any more than what's implemented in the code.